File: 351001/orsik/common/src/main/java/com/sergey/orsik/dto/response/CommentResponseTo.java
package com.sergey.orsik.dto.response;

import com.sergey.orsik.dto.CommentState;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CommentResponseTo {

    private Long id;
    private Long tweetId;
    private Long creatorId;
    private String content;
    private Instant created;
    private CommentState state;
}

File: 351003/Soika/entities-app/publisher/src/main/kotlin/com/example/entitiesapp/dto/CommentResponseTo.kt
package com.example.entitiesapp.dto

import java.io.Serializable

data class CommentResponseTo(
    val id: Long = 0,
    val content: String = "",
    val storyId: Long = 0,
    val country: String = "",
    val state: CommentState = CommentState.PENDING
) : Serializable